src/test/regress/pg_regress_multi.pl

```diff
@@ -56,6 +60,10 @@ ()
 my %functions = ();
 my %operators = ();
 my $valgrind = 0;
+my $valgrind_path = "valgrind";
+my $valgrind_log_file = "valgrind_test.log";
+my $pg_ctl_timeout = undef;
```

Comment:
That's a good point. Why did we update that line? Isn't it better to check if `$valgrind` is defined?